toolkit/offchain/src/csl.rs

```diff
@@ -140,6 +145,17 @@ pub(crate) fn convert_cost_models(m: &PlutusCostModels) -> Costmdls {
 	mdls
 }
 
+pub(crate) fn convert_reference_script_costs(
+	costs: &ReferenceScriptsCosts,
+) -> Result<UnitInterval, JsError> {
+	let r = Ratio::from_float(costs.base).unwrap();
```

Comment:
Please use Ratio<u64> and dont' go through strings below. Take numerator/denominator as u64 and use into()

Reply:
done